assets/js/modules/reader-revenue-manager/components/dashboard/ReaderRevenueManagerSetupCTABanner.js

```diff
@@ -169,136 +109,81 @@ function ReaderRevenueManagerSetupCTABanner( { Widget, WidgetNull } ) {
 		);
 	}
 
-	if ( ! showBanner ) {
-		return <WidgetNull />;
+	// TODO Remove this hack
+	// We "incorrectly" pass true to the `skipHidingFromQueue` option when dismissing this banner.
+	// This is because we don't want the component removed from the DOM as we have to still render
+	// the `AdminMenuTooltip` in this component. This means that we have to rely on manually
+	// checking for the dismissal state here.
+	if ( isCTADismissed || ! dismissedPromptsLoaded ) {
+		return null;
 	}
 
-	return (
-		<div className="googlesitekit-widget-context">
-			<Grid className="googlesitekit-widget-area">
-				<Row>
-					<Cell size={ 12 }>
-						<Widget
-							noPadding
-							className="googlesitekit-setup-cta-banner googlesitekit-reader-revenue-manager-setup-cta-widget"
-						>
-							<Grid collapsed>
-								<Row>
-									<Cell
-										smSize={ 12 }
-										mdSize={ 8 }
-										lgSize={ 6 }
-										className="googlesitekit-setup-cta-banner__primary-cell"
-									>
-										<h4 className="googlesitekit-setup-cta-banner__title">
-											{ __(
-												'Grow your revenue and deepen reader engagement',
-												'google-site-kit'
-											) }
-										</h4>
-										<div className="googlesitekit-setup-cta-banner__description">
-											<p>
-												{ createInterpolateElement(
-													__(
-														'Turn casual visitors into loyal readers and earn more from your content with voluntary contributions, surveys, newsletter sign-ups and reader insight tools. <a>Learn more</a>',
-														'google-site-kit'
-													),
-													{
-														a: (
-															<Link
-																href={
-																	readerRevenueManagerDocumentationURL
-																}
-																external
-																aria-label={ __(
-																	'Learn more about reader revenue manager',
-																	'google-site-kit'
-																) }
-															/>
-														),
-													}
-												) }
-												<br />
-												<br />
-												{ __(
-													'* Support for subscriptions coming soon',
-													'google-site-kit'
-												) }
-											</p>
-										</div>
+	const getBannerSVG = () => {
+		if ( breakpoint === BREAKPOINT_SMALL ) {
+			return SetupMobileSVG;
+		}
+
+		if ( breakpoint === BREAKPOINT_TABLET ) {
+			return SetupTabletSVG;
+		}
```

Comment:
This doesn't really need to be a function, but a map which could be defined outside of the component.

e.g.
```js
const breakpointSVGMap = {
	[ BREAKPOINT_SMALL ]: SetupMobileSVG,
	// ...
}
// In component...
SVG={ breakpointSVGMap[ breakpoint ] || SetupSVG };
```